test/game.tests/Tests.fs

```diff
@@ -18,11 +18,15 @@ let ``Parse and serialize returns same board except the annoying first newline``
 [<Fact>]
 let parse_wall_returns_wall () = 
     let board = "#"
-    Assert.True([['#']] =  game.parseBoard(board))
+    let expectedBoard = Map<int*int,Char> ([(0,0),'#'])
```

Comment:
Had a few tests that was testing internals, helpful when refatoring. Couldhave had more on deserialize/serialize, but that was easy when i had char list list and became harder only now.